scaling_performance/optimizing_storage.adoc

```diff
@@ -30,22 +30,21 @@ a|* Presented to the operating system (OS) as a block device
 bypassing the file system
 * Also referred to as a Storage Area Network (SAN)
 * Non-shareable, which means that only one client at a time can mount an endpoint of this type
-| CNS/CRS GlusterFS <1> iSCSI, Fibre Channel, Ceph RBD, OpenStack Cinder, AWS EBS <1>, Dell/EMC Scale.IO,
-VMware vSphere Volume, GCE Persistent Disk <1>, Azure Disk
+| CNS/CRS GlusterFS (1) iSCSI, Fibre Channel, Ceph RBD, OpenStack Cinder, AWS EBS (1), Dell/EMC Scale.IO,
+VMware vSphere Volume, GCE Persistent Disk (1), Azure Disk
 
 |File
 a| * Presented to the OS as a file system export to be mounted
 * Also referred to as Network Attached Storage (NAS)
 * Concurrency, latency, file locking mechanisms, and other capabilities vary widely between protocols, implementations, vendors, and scales.
-| CNS/CRS GlusterFS <1>, RHEL NFS, NetApp NFS <2>, Azure File, Vendor NFS, Vendor GlusterFS <3>, Azure File, AWS EFS
+| CNS/CRS GlusterFS (1), RHEL NFS, NetApp NFS (2), Azure File, Vendor NFS, Vendor GlusterFS (3), Azure File, AWS EFS
 
 | Object
 a| * Accessible through a REST API endpoint
 * Configurable for use in the {product-title} Registry
 * Applications must build their drivers into the application and/or container.
-| CNS/CRS GlusterFS <1>, Ceph Object Storage (RADOS Gateway), OpenStack Swift, Aliyun OSS, AWS S3, Google Cloud Storage, Azure Blob Storage, Vendor S3 <3>, Vendor Swift <3>
+| CNS/CRS GlusterFS (1), Ceph Object Storage (RADOS Gateway), OpenStack Swift, Aliyun OSS, AWS S3, Google Cloud Storage, Azure Blob Storage, Vendor S3 (3), Vendor Swift (3)
```

Comment:
The formatting of <1>, <2>, <3> is actually correct. I think we just need to add an "a" before the cell to get this to render properly. For example:

    a|CNS/CRS GlusterFS <1>, RHEL NFS, NetApp NFS <2>, Azure File, Vendor NFS, Vendor GlusterFS <3>, Azure File, AWS EFS

Give that a try and see if it builds better.